doc/flux_to_redux.md

```diff
@@ -0,0 +1,134 @@
+# Flux to Redux Transition Guide
+> A document explaining the fundamentals of transitioning a Flux architecture to a Redux architecture.
+---
+* __[Goal](#goal)__
+* __[Do I have to transition?](#do-i-have-to-transition-to-redux)__
+* __[Influx Architecture](#influx-architecture)__
+    * [Should You Use It](#should-you-implement-an-influx-architecture-during-the-transition?)
+        * [Advantages](#advantages)
+        * [Disadvantes](#disadvantages)
+* __[Example](#example-explained)__
+    * [Beginning Flux Architecture](#beginning-flux-architecture)
+    * [Getting to Influx Architecture](#getting-to-influx-architecture)
+        * [Influx Components](#influx-components)
+        * [Influx Stores](#influx-stores)
+        * [Example Specific Explanations](#example-specific-explanations)
+    * [Getting to Redux Architecture](#getting-to-redux-architecture)
+        * [Redux Stores](#redux-stores)
+        * [Redux Components](#redux-components)
+
+## Goal
+The goal of this document is explain major elements of transitioning to Flux from Redux. This includes the introduction of an Influx architecture (which means your state management system will be... in flux), a simple and advanced transition example, and specific steps to navigate the transition.
```

Comment:
![uhh-no](https://user-images.githubusercontent.com/41018730/71044908-4203b300-20f0-11ea-9f31-2b9fd7c32c16.gif)
```suggestion
The goal of this document is explain major elements of transitioning from Flux to Redux. This includes the introduction of an Influx architecture (which means your state management system will be... in flux), a simple and advanced transition example, and specific steps to navigate the transition.
```

Reply:
This is actually a doc to get people _to_ Flux, Keal :trollface:

Reply:
then you have a hell of alot MORE issue in other places, Joe :trollface:

Reply:
Ah shoot.. yeah, you're right. I guess I'll go and fix everything to go from redux _to_ flux..

Reply:
lol im sitting here looking at all gregs feedback and the fact that the first sentence had that issue...
![see-but-dont-observe](https://user-images.githubusercontent.com/41018730/71045149-264cdc80-20f1-11ea-9c12-54cccd8e3bb4.gif)
lulz jk @greglittlefield-wf i love you

Reply:
Hahahah this is why multiple reviewers are good